transmit-lib/src/cmr/transmit/metadata_db.clj

```diff
@@ -331,7 +331,15 @@
                                 :throw-exceptions false
                                 :headers (ch/context->http-headers context)}))
         status (int (:status response))
-        body (json/decode (:body response))
+        ;; For CMR-4841 - log the first 255 characters of the response body if
+        ;; the parsing of the html throws exception. 
+        response-body (:body response)
+        response-body-logged (subs response-body 0 (min 255 (.length response-body)))
```

Comment:
The standard way of getting sizing info in Clojure is `count`; did you try that instead of the `.length` Java method? It may be that count won't work on a byte stream? Dunno ... would have to check. This is one of the reasons I'd like to see a test ...

Reply:
response-body should be a string.  I have seen both count and .length, I just picked one, for no reason :)  I will change it to count if preferred.